src/core/core/model/word_list.py

```diff
@@ -307,3 +317,16 @@ def to_word_list_dict(self) -> dict[str, Any]:
 
     def to_entry_dict(self) -> dict[str, Any]:
         return {"value": self.value, "category": self.category}
+    
+    @classmethod
+    def load_multiple(cls, entries_data: list[dict]) -> list["WordListEntry"]:
+        if not entries_data:
+            return []
+        return [
+            cls(
+                value=entry.get('value'),
+                category=entry.get('category', 'Uncategorized'),
+                description=entry.get('description', '')
+            )
+            for entry in entries_data
+        ]
```

Comment:
I expect this to become obsolete as excluding the for loop, reusing code from [base_model.py](https://github.com/taranis-ai/taranis-ai/blob/master/src/core/core/model/base_model.py#L84) has the same functionality.

example:
```
    @classmethod
    def load_multiple(cls, entries_data: list[dict]) -> list["WordListEntry"]:
        return [cls.from_dict(entry) for entry in entries_data] if entries_data else []
```
but this is already in the BaseModel class almost identical.